### DISCUSSION TITLE
DAC codelists

### DISCUSSION THREAD

**Discussion by bill-anderson** [devinit/ddw-analyst-ui#427] [General]
Currently the IATI ETL brings in the DAC 3 Sector name and Finance Type Name. It doesn't bring in Sector Name or DAC Aid Type name. 

I propose we standardise our approach as follows.

- ETL should processs descriptions of IATI codes **_but not_** DAC codes.
- All DAC code lists should be available as separate tables
- Codelist tables should be named in a standard format, such as "Ref - DAC Sector Codes"
- Table column headers should not just be "Code ... Name" but e.g. "Finance Type Code ... Finance Type Name"

(This will mean that whenever the DAC changes their codes we only need to update a single process)

**Comment by k8hughes** [comment]
@bill-anderson what do we want @Duncan-Knox  to do on this?  If we can make the TOR for him nice and clear I can speak to him about when he would have time to deliver?

**Comment by k8hughes** [comment]
I am going to make this a discussion till there is more detail.
